bee-api/Cargo.toml

```diff
@@ -11,3 +11,17 @@ keywords = ["iota", "tangle", "bee", "framework", "api"]
 homepage = "https://www.iota.org"
 
 [dependencies]
+bee-common = { git = "https://github.com/iotaledger/bee.git" }
+bee-crypto = { path = "../bee-crypto" }
+bee-protocol = { path = "../bee-protocol" }
+bee-transaction = { path = "../bee-transaction" }
+bee-tangle = { path = "../bee-tangle" }
+
+async-std = { version = "1.6.2", features = ["attributes", "tokio02"] }
+async-trait = "0.1.36"
+bee-ternary = { git = "https://github.com/iotaledger/bee.git" }
```

Comment:
please move this up to its bro's and sis' :trollface:

Reply:
moved it up, see commit